app/controllers/development_controller.rb

```diff
@@ -274,6 +281,15 @@ def test_discussion
     @test_discussion
   end
 
+  def test_discussions
+    100.times do
+      Discussion.create!(title: Faker::Company.bs,
```

Comment:
As a side note, I've noticed we're basically duplicating FactoryGirl stuff in the development controller, do you think there might be a way to include those methods here, so this could just be

```
100.times { create(:discussion, group: group }
```
?